index.js

```diff
@@ -391,6 +391,76 @@ function prepareObjectStackTrace (obj, stack) {
 /**
  * Return a wrapped function in a deprecation message.
  */
+function createDeprecatedFn(fn, deprecate, message, site) {
+  switch(fn.length) {
```

Comment:
Without a default in the switch, wrapping functions with more than 10 arguments will silently return undefined instead of a wrapped function.